test/quantization/core/test_quantized_module.py

```diff
@@ -236,6 +239,9 @@ def _test_conv_api_impl(
             groups, kernel_size, stride, padding, padding_mode, dilation,
             X_scale, X_zero_point, W_scale, W_zero_point, Y_scale, Y_zero_point,
             use_bias, use_fused, use_channelwise):
+        # ONEDNN only supports symmetric quantization of weight
+        if torch.backends.quantized.engine == 'onednn' and not all(zp == 0 for zp in W_zero_point):
```

Comment:
if we select scale/zero_point based on qengine we won't need this check

Reply:
Do you mean to select proper weight scale/zp by `if ... else ....` in each unit test?

Reply:
yes, this function is called from: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_module.py#L387, we can generate the scale/zp based on the engine. the current implementation will just skip the check I think